CHANGELOG.md

```diff
@@ -4,9 +4,13 @@
 
 *February 11, 2021*
 
-Special thanks to external contributors on this release: @yayajacky, @odidev, @laniehei, and @c29r3!
+This release includes a fix for a memory leak introduced in the 0.34 release series (see #6068, below). 
```

Comment:
it's present in v0.33 too, so it wasn't introduced in 0.34

Reply:
in fact, it's present since evidence package came to life

Reply:
Ah, thanks - will correct.